modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptNestjsClientCodegen.java

```diff
@@ -0,0 +1,603 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
```

Comment:
```suggestion
```

Reply:
👍  although not sure why i should remove this a lot if not all the other codegens have this.